### ISSUE TITLE
Update official winget repository as part of the build toolchain

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-windows#155]
Windows now has an official commandline package manager, `winget` - see https://docs.microsoft.com/en-us/windows/package-manager/winget/ . It's similar to chocolatey, but it is intended to be the official package manager for Windows 10. It is open source, and the manifest Repo is [hosted on GitHub](https://github.com/microsoft/winget-pkgs/).

I was just about to submit the manifest for Kiwix JS Windows UWP app to it, when I found by strange coincidence that a Chinese developer had made a submission an hour or so earlier -- weird, right? In any case, that dev had made an error, because he had used the permalink, whereas we need explicit and predicatably versioned package URLs for the repo. This would have created an install error as soon as a new package was created, with invalid SHA256 etc.

I've been in touch, and that dev has closed his submission, and I've opened one with a versioned package, which has passed tests and is awaiting review. See https://github.com/microsoft/winget-pkgs/pull/16135. 

So I now need to ensure that `wincreate update` is incorporated into the build chain for the UWP app. This tool can update the manifests automatically and submit them to the winget repo.

**Comment by denelon** [comment]
@Jaifroid I commented in one of the Windows Package Manager repositories, but I wanted to follow up here. We have built a CI/CD tool https://github.com/microsoft/winget-create to help with this process. It is an early preview, but it might help. We would love your feedback, but feel free to integrate any way you see fit. Thank you for your willingness to support your customers with the Windows Package Manager.

**Comment by Jaifroid** [comment]
OK, thanks @denelon , I'll provide feedback once I've resolved this issue (using `winget-create` for sure).

**Comment by Jaifroid** [comment]
@denelon I've now used `wingetcreate update` a couple of times with my new releases. It's a very good tool, and it's great that it creates the PR on winget-pkgs staight away.  

I've incorporated a run of `wingetcreate update` into my `Create-DraftRelease.ps1` PowerShell script. Currently I run this interactively from the VS Code commandline. However, I've added switches that will allow this to be automated with GitHub Actions, using the same script, on release of a new tag, in the future.

It might be useful if your documentation stated clearly that the [CI example given](https://github.com/microsoft/winget-create#using-windows-package-manager-manifest-creator-in-a-cicd-pipeline) works with GitHub Actions and not only with Azure Pipelines, but I'm sure it's easy to adapt. Thank you for the tool!
